src/main/java/org/manifold/compiler/back/digital/VHDLCodeGenerator.java

```diff
@@ -32,7 +26,8 @@
   private static String newline = System.getProperty("line.separator");
```

Comment:
This is sort of inconsistent to be using line.separator here, File.separator elsewhere, "/" in tests.